Neos.ContentRepository.Core/Classes/Feature/Common/TetheredNodeInternals.php

```diff
@@ -123,25 +125,40 @@ protected function createEventsForMissingTetheredNode(
                         $creationOriginDimensionSpacePoint = $rootGeneralizationOrigin;
                     }
                 }
-                return Events::fromArray($events);
+                $events = Events::fromArray($eventsArray);
+            } else {
+                $events = Events::with(
+                    event: new NodeAggregateWithNodeWasCreated(
+                        workspaceName: $contentGraph->getWorkspaceName(),
+                        contentStreamId: $contentGraph->getContentStreamId(),
+                        nodeAggregateId: $tetheredNodeAggregateId,
+                        nodeTypeName: $tetheredNodeTypeDefinition->nodeTypeName,
+                        originDimensionSpacePoint: $originDimensionSpacePoint,
+                        succeedingSiblingsForCoverage: InterdimensionalSiblings::fromDimensionSpacePointSetWithoutSucceedingSiblings(
+                            dimensionSpacePointSet: $parentNodeAggregateCoverageByOccupant
+                        ),
+                        parentNodeAggregateId: $parentNodeAggregateId,
+                        nodeName: $tetheredNodeTypeDefinition->name,
+                        initialPropertyValues: $defaultProperties,
+                        nodeAggregateClassification: NodeAggregateClassification::CLASSIFICATION_TETHERED,
+                        nodeReferences: SerializedNodeReferences::createEmpty(),
```

Comment:
I wonder about the named parameter. I guess this is rather your preferred coding style than actually needed? Not sure if we already discussed this in past, but IMHO we should than.